cd/Jenkinsfile_utils.groovy

```diff
@@ -19,16 +19,16 @@
 
 // Triggers a downstream jenkins job responsible for building, testing
 // and publishing all the variants for a particular 'job_type'.
-// The 'job_type' should be the name of the directory that contains the 
-// 'Jenkins_pipeline.groovy' file and has the pipeline definition for the 
+// The 'job_type' should be the name of the directory that contains the
+// 'Jenkins_pipeline.groovy' file and has the pipeline definition for the
 // artifact (docker image, binary, pypi or maven package, etc.) that should
 // be published.
 
 STATE_UPDATE="State Update"
 
-def trigger_release_job(job_name, job_type, mxnet_variants) {
+def trigger_release_job(cd_release_job, job_name, job_type, mxnet_variants) {
```

Comment:
Curious, why do we pass the cd_release_job instead of using the env var?

Reply:
CD_RELEASE_JOB_NAME is actually a parameter of the CD pipeline, so this is how it's supposed to be used. It is also set as a global env variable, which can potentially cause some issues.
For reference, it is addressed in master by this PR https://github.com/apache/incubator-mxnet/pull/17775